src/Sulu/Bundle/AdminBundle/Resources/js/components/Block/tests/Block.test.js

```diff
@@ -20,96 +20,100 @@ test('Render an expanded block with a multiple types', () => {
             types={{'type1': 'Type1', 'type2': 'Type2'}}
         >
             Some block content
-        </Block>
-    )).toMatchSnapshot();
+        </Block>);
+
+    expect(container).toMatchSnapshot();
 });
 
 test('Render an block without dragHandle or collapse or expand button', () => {
-    expect(render(
+    const {container} = render(
         <Block expanded={true}>
             Some block content
         </Block>
-    )).toMatchSnapshot();
+    );
+    expect(container).toMatchSnapshot();
 });
 
 test('Render a collapsed block', () => {
-    expect(render(
+    const {container} = render(
         <Block expanded={false} icons={['su-eye', 'su-people']} onCollapse={jest.fn()} onExpand={jest.fn()}>
             Some block content
         </Block>
-    )).toMatchSnapshot();
+    );
+    expect(container).toMatchSnapshot();
 });
 
 test('Do not show type dropdown if only a single type is passed', () => {
-    const block = shallow(
+    render(
         <Block expanded={true} onCollapse={jest.fn()} onExpand={jest.fn()} types={{'type': 'Type'}}>
             Some block content
         </Block>
     );
 
-    expect(block.find('SingleSelect')).toHaveLength(0);
+    const element = screen.queryByLabelText('select');
```

Comment:
We should go herre with `getByText` and check for the `displayValue` of the select.

Reply:
This regards the discussion of the `aria-label` problem. Sometimes it's hard to query items because (as you mentioned) there is literally no relevant information about a DOM element for RTL. If that is the case one would have to navigate though the tree (e.g. using `.parentElement`, `.children[0]`) which is not allowed by ESlint. Even simply accessing elements by `.getElementsbyClassName()` if forbidden by the linter. The example below passes the test but not the linter. Any suggestions?
![image](https://user-images.githubusercontent.com/54234287/181482737-86a1ad7a-4151-40b0-849a-36420ac0ce00.png)

Reply:
I'm personally fine to add in the test a `// eslint-disable-next-line the-violating-rule` if it make the test easier to understand by using this selector.

Reply:
But as said you can also add a comment and we will check how we can improve it to make this things easier.